ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -369,16 +370,11 @@ private List<DiscoveryPeer> getPeersFromNeighborsPacket(final Packet packet) {
         .collect(Collectors.toList());
   }
 
-  private boolean addToPeerTable(final DiscoveryPeer peer) {
+  private boolean addBondedPeerToPeerTable(final DiscoveryPeer peer) {
```

Comment:
what about addToPeerTableAndBond()

Reply:
The state of the peer is already bonded or will be changed to bonded in this method